### DISCUSSION TITLE
AirPlay support?

### DISCUSSION THREAD

**Discussion by recycle-ed** [TortugaPower/BookPlayer#865] [Q&A] [answered]
Does BookPlayer support sending audio to AirPlay devices? I don't see that as an option now. If not, it be possible/difficult to add this feature?

**Comment by recycle-ed** [comment, marked as answer]
I was able to resolve this. The airplay icon was sitting right there in front of me.

**Reply by GianniCarlo** [reply]
I'm glad you found it 🙌, depending on the artwork, it may not be as obvious 😓 in the player screen, for future reference, it is also accessible from either the lock screen now playing screen, or the control center now playing bubble 👍
